src/main/java/org/folio/rest/impl/RtacResourceImpl.java

```diff
@@ -250,17 +253,21 @@ private String getVolume(JsonObject item) {
     // |chronology volume            |(<volume>)                  |
     // |chronology                   |(<chronology>)              |
 
-    final StringJoiner sj = new StringJoiner(" ", "(", ")");
-
-    return enumeration
-        .map(e -> chronology
-          .map(c -> sj.add(e).add(c).toString())
-          .orElse(sj.add(e).toString()))
-        .orElseGet(() -> volume
-            .map(v -> sj.add(v).toString())
-            .orElseGet(() -> chronology
-                .map(c -> sj.add(c).toString())
-                .orElse(null)));
+    final StringJoiner sj = new StringJoiner(" ", "(", ")").setEmptyValue("");
+
+    if (isNotBlank(enumeration)) {
```

Comment:
Thanks for rewriting this block so java novices like me can follow the code ;-)

Reply:
Yeah, it was ridiculous as previously written. My bad...